src/framework/render.js

```diff
@@ -1,21 +1,26 @@
 /** @jsx createElement */
 /*** @jsxFrag createFragment */
 import { createElement } from './element';
-
-let Component, Target;
+import { current } from './hooks';
 
 /**
  * Renders a component and attaches it to the target DOM element
- * @param componentFunction - Component function or class
+ * @param Component - function or class
  * @param target - DOM element to attach component to
  */
-function renderApp(componentFunction = null, target = null) {
-  // Memorize parameters to enable re-render when `renderApp` gets called without arguments
-  if (componentFunction) Component = componentFunction;
-  if (target) Target = target;
-  Target.innerHTML = '';
-  // Ensure that the component gets rebuilt
-  Target.appendChild(<Component />);
+
+let intervalID;
+const RERENDER_FREQUENCY = 300;
+
+export function render(Component, target) {
+  intervalID = setInterval(() => {
```

Comment:
we could mention also this link in comments
https://css-tricks.com/snippets/javascript/replacements-setinterval-using-requestanimationframe/
for those who wants more performant solution